onnxruntime/core/framework/error_code.cc

```diff
@@ -26,6 +30,13 @@ inline OrtStatus* NewStatus(size_t clen) {
   if (buf == nullptr) return nullptr;  // OOM. What we can do here? abort()?
   return new (buf) OrtStatus;
 }
+
+inline void DeleteStatus(OrtStatus* ort_status) {
+#if defined(_MSC_VER) && !defined(__clang__)
+#pragma warning(suppress : 26409)
+#endif
```

Comment:
Should this push/pop the warning?

Reply:
apparently suppress really does only suppress it for the next line, which is `#endif`. oops.